dbt_project.yml

```diff
@@ -5,31 +5,34 @@ require-dbt-version: [">=1.0.0", "<2.0.0"]
+    app: "{{ source('apple_store', 'app') }}"
+    app_store_device: "{{ source('apple_store', 'app_store_source_type_device_report') }}"
+    usage_device: "{{ source('apple_store', 'usage_source_type_device_report') }}"
+    downloads_device: "{{ source('apple_store', 'downloads_source_type_device_report') }}"
 
-    app_store_territory_report: "{{ source('apple_store', 'app_store_territory_source_type_report') }}"
-    app_units_territory_report: "{{ source('apple_store', 'app_units_territory_source_type_report') }}"
-    downloads_territory_report: "{{ source('apple_store', 'downloads_territory_source_type_report') }}"
-    usage_territory_report: "{{ source('apple_store', 'usage_territory_source_type_report') }}"
+    app_store_territory: "{{ source('apple_store', 'app_store_territory_source_type_report') }}"
+    downloads_territory: "{{ source('apple_store', 'downloads_territory_source_type_report') }}"
+    usage_territory: "{{ source('apple_store', 'usage_territory_source_type_report') }}"
 
-    app_store_platform_version_report: "{{ source('apple_store', 'app_store_platform_version_source_type_report') }}"
-    app_units_platform_version_report: "{{ source('apple_store', 'app_units_platform_version_source_type_report') }}"
-    crashes_platform_version_report: "{{ source('apple_store', 'crashes_platform_version_report') }}"
-    downloads_platform_version_report: "{{ source('apple_store', 'downloads_territory_source_type_report') }}"
-    usage_platform_version_report: "{{ source('apple_store', 'usage_territory_source_type_report') }}"
+    app_store_platform_version: "{{ source('apple_store', 'app_store_platform_version_source_type_report') }}"
+    crashes_platform_version: "{{ source('apple_store', 'crashes_platform_version_device_report') }}"
+    downloads_platform_version: "{{ source('apple_store', 'downloads_platform_version_source_type_report') }}"
+    usage_platform_version: "{{ source('apple_store', 'usage_platform_version_source_type_report') }}"
 
-    crashes_app_version_report: "{{ source('apple_store', 'crashes_app_version_report') }}"
-    usage_app_version_report: "{{ source('apple_store', 'usage_app_version_source_type_report') }}"
+    crashes_app_version: "{{ source('apple_store', 'crashes_app_version_device_report') }}"
+    usage_app_version: "{{ source('apple_store', 'usage_app_version_source_type_report') }}"
 
     sales_account: "{{ source('apple_store', 'sales_account') }}"
-    sales_subscription_event_summary: "{{ source('apple_store', 'sales_subscription_event_summary') }}"
+    sales_subscription_events: "{{ source('apple_store', 'sales_subscription_event_summary') }}"
     sales_subscription_summary: "{{ source('apple_store', 'sales_subscription_summary') }}"
 
+    apple_store__subscription_events:
+    - 'Renew'
+    - 'Cancel'
+    - 'Subscribe'
+    
 models:
-  apple_store_source:
+  apple_search_ads_source:
     materialized: table
     +schema: stg_apple_store
```

Comment:
did we decide to go with `apple_store_source` instead of `stg_apple_store` for the schema suffix?